### PR TITLE
[hu] translation

### PR DESCRIPTION
**Pull request by kzolee1**
Just started a Codeigniter 4 project and realized that it has no Hungarian translation, like ci3 had.

### CODE REVIEW

**Comment by paulbalandan** [comment]
Hi! Thank you for the PR! However, you are missing the unit tests to check if you got all complete. Please read on the [Contributing Guide](https://github.com/codeigniter4/translations/blob/develop/CONTRIBUTING.md#extending-unit-tests)

**Comment by kzolee1** [comment]
Thank You for opening my eyes :) The unit tests have been made, some minor modifications were required, now it passes.

**Review by kenjis — APPROVED** [review]

**Review by paulbalandan — COMMENTED** [review]
Almost there! Thanks for the changes. Some minor comments to finish up.

- Add `hu` to the table in README
- Remove `hu` from the MISSING.rst
- Alphabetically arrange `hu` in `AbstractTranslationTestCase`

File: tests/Language/AbstractTranslationTestCase.php [hunk 1]
```diff
@@ -66,6 +66,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         VietnameseTranslationTest::class         => 'vi',
         SimplifiedChineseTranslationTest::class  => 'zh-CN',
         TraditionalChineseTranslationTest::class => 'zh-TW',
+        HungarianTranslationTest::class          => 'hu',
```
[the remark this answers is not in the document]
Comment (kzolee1) [reply tests/Language/AbstractTranslationTestCase.php]: Requested modifications have been made.

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
Thank you, @kzolee1

**Pull request merged**